Project.toml

```
@@ -15,7 +15,7 @@ Static = "aedffcd0-7271-4cad-89d0-dc628f76c6d3"
 
 [compat]
 BayesBase = "1.3"
-ExponentialFamily = "1.4"
+ExponentialFamily = ">= 1.4.3"
```

This is a bad compat entry, better to leave it as "1.4.3". Please read the differences [here](https://pkgdocs.julialang.org/v1/compatibility/#Compatibility). In short `>=` allows breaking versions.
```suggestion
ExponentialFamily = "1.4.3"
```
After that can be merged!